cmd/ui/src/components/SettingsMenu.tsx

```diff
@@ -139,11 +149,24 @@ const SettingsMenu: React.FC<Props> = ({ anchorEl, handleClose }) => {
                     onClick={() => openInNewTab('https://bloodhoundenterprise.io/our-solution/')}
                     data-testid='global_header_settings-menu_nav-checkout-BHE'>
                     <ListItemIcon>
-                        <EnterpriseIcon fill={'#000000'} width='1rem' height='1rem' />
+                        <EnterpriseIcon fill={theme.palette.color.primary} width='1rem' height='1rem' />
                     </ListItemIcon>
                     <ListItemText primary='BloodHound Enterprise' />
                 </MenuItem>
 
+                <FeatureFlag
```

Comment:
Currently, if the `dark_mode` flag is not found, this menu item will display the string "Error". If possible, it would be preferable to hide this element entirely:
<img width="230" alt="Screenshot 2024-07-22 at 1 04 55 PM" src="https://github.com/user-attachments/assets/a2a45d59-7f83-4361-92ec-9be00d244688">

Reply:
Great find! updated!